### PR TITLE
🐛Fix subdomain handling in mgid widgets

### PR DESCRIPTION
**Pull request by velichkin**
Mgid widget wouldn't work if first or second char of data.publisher is not alphanumeric.

### CODE REVIEW

File: ads/mgid.js [hunk 1]
```diff
@@ -28,9 +28,11 @@ export function mgid(global, data) {
 
   document.body.appendChild(scriptRoot);
 
+  const publisherStr = data.publisher.replace(/[^A-z0-9]/g, '');
```
Comment (choumx) [inline ads/mgid.js]: Consider adding a comment explaining this.

**Comment by velichkin** [comment]
@choumx I've made changes. Is it ok now?

**Comment by velichkin** [comment]
@choumx @lannka

**Pull request merged**